spec/qiita/markdown/processor_spec.rb

```diff
@@ -759,20 +754,7 @@
         end
 
         it "generates footnotes elements" do
-          should eq <<-HTML.strip_heredoc
-            <p><sup id="fnref1"><a href="#fn1" title="test">1</a></sup></p>
-
-            <div class="footnotes">
-            <hr>
-            <ol>
-
-            <li id="fn1">
-            <p>test <a href="#fnref1">↩</a></p>
-            </li>
-
-            </ol>
-            </div>
-          HTML
+          should include('class="footnotes"')
```

Comment:
Change to just test genarating footnotes element because some attributes in footnotes will be removed under strict context.